.changesets/docs_geal_document_request_lifecycle.md

```diff
@@ -1,5 +1,9 @@
 ### Document the request lifecycle ([PR #3391](https://github.com/apollographql/router/pull/3391))
 
-This adds an in depth documentation of the entire request lifecycle, which services exist in the router, the request and response types they use, and where plugins can attach themselves
+This adds in-depth documentation of:
```

Comment:
If a sentences includes a list of 3+ items, it's worth considering if it would be clearly as a list.